unstructured/ingest/v2/cli/base/cmd.py

```diff
@@ -34,6 +34,10 @@ class BaseCmd(ABC):
     def cmd_name_key(self):
         return self.cmd_name.replace("-", "_")
 
+    @property
+    def cli_cmd_name(self):
+        return self.cmd_name.replace("_", "-")
```

Comment:
Is this in case people use a CLI arg like `--collection_name` accidentally? Or is this something that happens with every CLI arg?

Reply:
Nah, this is when the connector id is something like `google_drive` to make the command name `google-drive`